src/app/containers/RadioSchedule/index.jsx

```diff
@@ -1,128 +1,38 @@
-import React, { useContext, useEffect, useState } from 'react';
-import 'isomorphic-fetch';
+import React, { useContext } from 'react';
 import { string } from 'prop-types';
-import RadioSchedule from '@bbc/psammead-radio-schedule';
-import moment from 'moment';
-import findLastIndex from 'ramda/src/findLastIndex';
-import propSatisfies from 'ramda/src/propSatisfies';
 import { ServiceContext } from '#contexts/ServiceContext';
+import { RequestContext } from '#contexts/RequestContext';
 import useToggle from '../Toggle/useToggle';
-import webLogger from '#lib/logger.web';
+import Canonical from './Canonical';
 
-const logger = webLogger();
+const getRadioScheduleEndpoint = service =>
+  `/${service}/bbc_${service}_radio/schedule.json`;
 
-const RadioScheduleContainer = ({ endpoint }) => {
+const RadioScheduleContainer = ({ radioScheduleEndpointOverride }) => {
   const { enabled } = useToggle('radioSchedule');
+  const { isAmp } = useContext(RequestContext);
   const {
-    radioSchedule: { hasRadioSchedule },
     service,
-    script,
-    dir,
-    timezone,
-    locale,
+    radioSchedule: { hasRadioSchedule },
```

Comment:
Minor comment, would it be safer to use `pathOr` here as well - we don't validate the service config so would guard against an error here